packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec

```diff
@@ -84,5 +84,8 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/spec
 
 %changelog
+* Mon Feb 10 2020 <John Mitsch> 0.0.1b2.dev01581272120-1
```

Comment:
Normally this is in the form of `My Name <myname@example.com>`. Interesting that Justin also didn't do this.